app/scripts/modules/core/src/presentation/horizontalTabs/HorizontalTabs.tsx

```diff
@@ -0,0 +1,53 @@
+import cx from 'classnames';
+import React from 'react';
+import './HorizontalTabs.less';
+
+export const TopNav = () => {};
```

Comment:
I am looking per commit, so I may be missing something. Can you explain why this is a noop and where this is used?

Reply:
Ooops